diff -du TxBR/setup.py TxBR_good/setup.py
--- TxBR/setup.py	2009-10-11 17:10:03.000000000 -0700
+++ TxBR_good/setup.py	2013-01-24 18:15:11.959612215 -0800
@@ -49,7 +49,7 @@
 
 preconstruction = Extension("txbr.prefil",
                     include_dirs = [imod_dir_include, txbr_dir_include],
-                    libraries = [ 'iimod', 'imod', 'cfshr', 'fftw3', 'fftw3f', 'cv', 'cxcore' ],
+                    libraries = [ 'iimod', 'imod', 'cfshr', 'fftw3', 'fftw3f'  ],
                     library_dirs = [ imod_dir_lib, ],
                     sources = ["txbr/txbr/filter/run_prefil.c","txbr/txbr/filter/filter_1D.c","txbr/txbr/filter/utilities.c","txbr/txbr/filter/utilities_base.c","txbr/txbr/filter/edgetaper_2D.c"],
                     extra_compile_args = ["-std=gnu99"]
